### DISCUSSION TITLE
Do you support caching by number of hits?

### DISCUSSION THREAD

**Discussion by AhmedAredah** [Yiling-J/theine#32] [General]
Do you support caching by number of hits? If the entry has lowest hits, there is higher chance of it being removed from the cache? (Not by time) Thanks

**Comment by Yiling-J** [comment]
@AhmedAredah You can think of Theine as an approximate LFU cache, which uses frequency to determine which entries to keep and which to evict when the cache is full. If I understand correctly, the `hits` here is same as frequency? If so, that’s already how Theine operates. If time isn’t a critical factor, you can set a very long TTL.
